f5_os_test/infrastructure.py

```diff
@@ -19,30 +19,40 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption("--bigip", action="store",
+    parser.addoption("--bigip-address", action="store",
                      help="BIG-IP hostname or IP address")
-    parser.addoption("--username", action="store", help="BIG-IP REST username",
+    parser.addoption("--bigip-username", action="store",
+                     help="BIG-IP REST username",
                      default="admin")
-    parser.addoption("--password", action="store", help="BIG-IP REST password",
+    parser.addoption("--bigip-password", action="store",
+                     help="BIG-IP REST password",
                      default="admin")
-    parser.addoption('--auth_address', action='store',
+    parser.addoption("--auth-address", action="store",
                      help="Keystone Authorization server name, or IP address.")
+    parser.addoption("--os-tenant-id", action="store",
+                     help="ID for keystone tenant.")
+    parser.addoption("--os-username", action="store",
+                     help="Openstack username.")
+    parser.addoption("--os-password", action="store",
+                     help="Openstack password.")
+    parser.addoption("--os-tenant-name", action="store",
+                     help="Openstack tenant name.")
 
```

Comment:
I'm mostly +1 on these changes, but I want to consider using the "netloc" alias for hostnames-or-addresses.  That is the convention the excellent urlparse library uses for the same purpose.   Since a 'hostname' will work in those fields, we should not really call them "addresses"....   (I know, I know, I started it....)